### PR TITLE
Final launch tweaks. Add license.

### PR DESCRIPTION
**Pull request by dbrehmer**

### CODE REVIEW

**Review by simusud — APPROVED** [review]

File: core_contracts/governance/governance.py [hunk 1]
```diff
@@ -75,8 +89,56 @@ def launchBalanced(self) -> None:
...
+        dex = self.create_interface_score(dex_address, DexInterface)
+        rewards.claimRewards()
+        baln_value = baln.balanceOf(self.address)
+        bnUSD_value = _baln_price * baln_value // 100  # BALN price is in pennies.
+        loans.depositAndBorrow('bnUSD', bnUSD_value)
+        bnUSD.transfer(dex_address, bnUSD_value, json_dumps({"method": "_deposit"}).encode())
+        baln.transfer(dex_address, baln_value, json_dumps({"method": "_deposit"}).encode())
+        dex.add(baln_address, bnUSD_address, baln_value, bnUSD_value)
+        name = 'BALN/bnUSD'
+        pid = 3
```
Comment (gedanziger) [inline core_contracts/governance/governance.py:138]: Instead, use the function:

```
getPoolId(self, _token1Address: Address, _token2Address: Address)
```

passing in the addresses of baln and bnUSD

File: core_contracts/governance/governance.py [hunk 2]
```diff
@@ -75,8 +89,56 @@ def launchBalanced(self) -> None:
...
+        amount = UNITS_PER_TOKEN * value // (price * 7)
+        staking.icx(value // 7).stakeICX()
+        loans.icx(self.icx.get_balance(self.address)).depositAndBorrow('bnUSD', amount)
+        bnUSD_value = bnUSD.balanceOf(self.address)
+        sICX_value = sICX.balanceOf(self.address)
+        bnUSD.transfer(dex_address, bnUSD_value, json_dumps({"method": "_deposit"}).encode())
+        sICX.transfer(dex_address, sICX_value, json_dumps({"method": "_deposit"}).encode())
+        dex.add(sICX_address, bnUSD_address, sICX_value, bnUSD_value)
+        name = 'sICX/bnUSD'
+        pid = 2
```
Comment (gedanziger) [inline core_contracts/governance/governance.py:115]: Instead, use the function:

getPoolId(self, _token1Address: Address, _token2Address: Address)
passing in the addresses of sICX and bnUSD

**Review by gedanziger — CHANGES_REQUESTED** [review]
This should work, but we can improve it slightly with this edit.

File: core_contracts/governance/governance.py (hunk 2 above)
Comment (dbrehmer) [reply core_contracts/governance/governance.py:115]: Nice. Thanks.

Comment (dbrehmer) [reply core_contracts/governance/governance.py]: Should be corrected now.

**Pull request merged**